pkg/terminal/console_windows.go

```diff
@@ -25,7 +25,7 @@ func setConsoleMode(handle windows.Handle, flags uint32) error {
 	var mode uint32
 	err := windows.GetConsoleMode(handle, &mode)
 	if err != nil {
-		return err
+		return nil // not a terminal
```

Comment:
Shouldn't we check for specific error here, like NotATerminal or something like rhat?

Reply:
Golang standard library doesn't think it's worth the bother, and I agree. The only thing about the handle the code here is interested in is whether it makes sense to call `SetConsoleMode` on it to turn on VT mode. If standard handles _are_ terminal handles but there is something else wrong with them that makes `GetConsoleMode` return an error, the terminal will just remain in dumb mode and the error will come up a bit later when trying to read or write to them. IMO this is reasonable behavior.